doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -6,131 +6,366 @@ Cite Test Guide
-      cd ets-wcs10.git
-      mvn install
 
 Run WFS 1.0 tests
 -----------------
 
 .. important::
 
-   Running WFS 1.0 tests require PostGIS to be installed on the system.
+   Running WFS 1.0 tests require PostgreSQL with PostGIS extension installed to be installed on the system.
 
 Requirements:
 ~~~~~~~~~~~~~
+
+- `GeoServer running`
+- teamengine
 - Posgresql
 - PostGIS
 
-#. Create a PostGIS user named "cite":
+#. Prepare the environment:
 
-   .. code:: SQL
+   - login in postgresql and create a user named "cite".
 
-      CREATEUSER cite
+   .. code:: sql
 
-#. Create a PostGIS databased named "cite", owned by the "cite" user:
+     createuser cite;
 
-   .. code:: SQL
+   - Create a database named "cite", owned by the "cite" user:
 
-      CREATEDB -T template_postgis -U cite cite
+   .. code:: sql
 
-#. Change directory to the ``citewfs-1.0`` data directory and execute the script ``cite_data.sql``:
+     createdb cite own by cite;
 
-   .. code:: shell
+   - enter to the database and enable the postgis extension:
 
-     psql -U cite cite < cite_data.sql
+   .. code:: sql
 
-#. Start GeoServer with the ``citewfs-1.0`` data directory. Example:
+    create extension postgis;
 
-   .. code:: shell
+   - Change directory to the citewfs-1.0 data directory and execute the script cite_data_postgis2.sql:
+
+   .. code-block:: shell
+
+    cd <root of geoserver repository>
+    psql -U cite cite < cite_data_postgis2.sql
+
+   - Start GeoServer with the citewfs-1.0 data directory. Example:
+
+   .. important::
+
+     If the postgresql server is not in the same host of the geoserver, you have to change the `<entry key="host">localhost</entry>` in the `datastore.xml` file, located inside each workspace directory. ex.
+
+     .. note::
+
+       <root of geoserver sources>/data/citewfs-1.0/workspaces/cgf/cgf/datastore.xml
+
+   .. code-block:: shell
 
-     cd <root of geoserver install>
-     export GEOSERVER_DATA_DIR=<root of geoserver sources>/build/cite/wfs11/citewfs-1.1
-     ./bin/startup.sh
+    cd <root of geoserver install>
+    export GEOSERVER_DATA_DIR=<root of geoserver sources>/data/citewfs-1.0
+    ./bin/startup.sh
 
-#. Change directory back to the cite tools and run the tests:
+#. Start the test:
 
    .. code:: shell
 
-     ant wfs-1.0
-     
+     make webUI
+
+#. Go to the browser and open the URL: http://localhost:8888/teamengine/
+
+   - after the site open, click on the **Sign in** button and enter the user and password. 
+
    With the following parameters:
 
-   #. ``Capabilities URL`` http://localhost:8080/geoserver/wfs?request=getcapabilities&service=wfs&version=1.0.0
+   #. ``Capabilities URL`` http://<ip.of.the.goserver>:8080/geoserver/wfs?request=getcapabilities&service=wfs&version=1.0.0
 
-   #. ``All`` tests included
+   #. ``Enable tests with multiple namespaces`` tests included
 
-      .. image:: tewfs-1_0.png
+      .. image:: ./image/tewfs-1_0.png
```

Comment:
After running the test the report seems missing. It's not true, but the output can be confusing.

In that paragraph of things to pay attention to, before the list of suites, tell developers that they should expect for some old test suites an output like:

![image](https://user-images.githubusercontent.com/325433/114408032-50a98f80-9ba9-11eb-83cc-87e703a4669d.png)

The first sentence is confusing, seems like one has to drop on the comand line and look at XML, but instead one can follow the link to the "detailed old test report" and then the report is there and it's usable:

![image](https://user-images.githubusercontent.com/325433/114408209-7a62b680-9ba9-11eb-8b19-3f8c7c493bd6.png)

Reply:
resolved in this commit https://github.com/randomorder/geoserver/pull/3/commits/36a49e75d9c99b3948a52a63f21cd60741c65e1b